### DISCUSSION TITLE
Links de instalacion para Git  y editores de texto.

### DISCUSSION THREAD

**Discussion by sebasvelasco353** [sebasvelasco353/FDP-2022_01#1] [Announcements]
Casi todos estos programas los pueden descargar e instalar como cualquier otro, recuerden que pueden elegir cualquier editor de texto de los que aparecen aqui u otro que uds prefieran, el que voy a usar yo durante las clases es VSCode.

- [Git **windows**](https://git-scm.com/download/wi)
- [Git **Mac**](https://git-scm.com/download/mac), Si tienen dudas sobre como instalar [Homebrew](https://brew.sh/) y Git pueden preguntarme en la clase y les ayudo.
- [VSCode](https://code.visualstudio.com/)
- [Atom](https://atom.io/)
- [Sublime Text](https://www.sublimetext.com/)

Cualquier duda que tengan recuerden que pueden tambien seguir esta discusion en el post y recibir ayuda.

**Comment by Agudelo02093** [comment]
Hola profe! por favor no olvides los videos que te pedimos en clase. Yo personalmente estoy aún un poco perdida con el tema. Gracias!

**Reply by sebasvelasco353** [reply]
Si, estoy trabajando en ellos para mandarselos.

**Reply by sebasvelasco353** [reply]
ya quedo en youtube, el [link](https://youtu.be/y2G9ePYV3C8) esta tambien en los anuncios.
